Microsoft.Azure.Cosmos/tests/Microsoft.Azure.Cosmos.Tests/ChangeFeed/NetworkAttachedDocumentContainerTests.cs

```diff
@@ -21,6 +22,28 @@ namespace Microsoft.Azure.Cosmos.Tests.ChangeFeed
     public class NetworkAttachedDocumentContainerTests
     {
         [TestMethod]
+        public async Task TestKeyRangeCacheRefresh()
+        {
+            const string resourceId = "resourceId";
+            Mock<CosmosClientContext> context = new Mock<CosmosClientContext>();
+            Mock<ContainerInternal> container = new Mock<ContainerInternal>();
+            container.Setup(c => c.ClientContext).Returns(context.Object);
+            container.Setup(c => c.GetCachedRIDAsync(It.IsAny<bool>(), It.IsAny<ITrace>(), It.IsAny<CancellationToken>())).ReturnsAsync(resourceId);
```

Comment:
nit: just to make sure it's doesn't force refresh the cache.
```suggestion
            container.Setup(c => c.GetCachedRIDAsync(false, It.IsAny<ITrace>(), It.IsAny<CancellationToken>())).ReturnsAsync(resourceId);
```